### DISCUSSION TITLE
Add ability to tag sprint when doing "quick create" of issues in Github Projects

### DISCUSSION THREAD

**Discussion by naunidh-tetrix** [community/community#137960] [Projects and Issues]
### Select Topic Area

Product Feedback

### Feature Area

Projects

### Body

Can you please include the ability to assign a sprint to issues at the point of creation itself? Right now, when we create an issue, the pop-up modal only allows assignment of labels and assignees, not sprints. Hence, we need to create the issue, click on it and then assign sprints which is cumbersome, especially when we are creating multiple new issues per sprint.

### Guidelines

- [X] I have read the above statement and can confirm my post is relevant to the GitHub feature areas [Issues](https://docs.github.com/en/issues/tracking-your-work-with-issues/about-issues) and/or [Projects](https://docs.github.com/en/issues/planning-and-tracking-with-projects/learning-about-projects/about-projects).

**Comment by TikoyX** [comment]
What do you mean when you say "assign sprint"? Do you mean a project?
I've never seen anything called "assign sprint" on GitHub

**Reply by naunidh-tetrix** [reply]
In Project settings, you can add an iteration (ie: Sprint). However, when you do "create issue" the pop-up modal has no option visible to tag that issue to a sprint/iteration. Only after the issue is created can you do so (versus at the point of creation itself). See attached screenshot for more info: 
<img width="773" alt="Missing Sprint" src="https://github.com/user-attachments/assets/51c47590-7a46-42c0-a72e-d4c079e080a6">

**Reply by TikoyX** [reply]
Oh thank you so much. I've never noticed this before.
I don't know if i think this is the optimal way for me to do sprints in GitHub, but it is interesting to see this take on it.
Unfortunately there is a LOT of functionality GitHub doesn't have, like the thing you suggested. I don't understand how so big a company can be so far behind on features

**Reply by naunidh-tetrix** [reply]
Same...hope they look into this soon!

**Reply by TikoyX** [reply]
Just all custom fields in general. It seems so stupid you have to go into an issue afterwards its on the project to set these values
